app/_hooks/useCoFollower.ts

```diff
@@ -0,0 +1,66 @@
+import { getFollower, getFollowing } from '@/_apis/getFollow';
+
+import { useQuery } from '@tanstack/react-query';
+
+interface UserInfo2 {
+  avatar_url: string;
+
+  login: string;
+  node_id: string;
+  organizations_url: string;
+  received_events_url: string;
+  repos_url: string;
+  site_admin: boolean;
+  starred_url: string;
+  subscriptions_url: string;
+  type: string;
+  url: string;
+}
+
+interface UserInfo {
+  avatar_url: string;
+  login: string;
+}
+const useGetCoFollower = (pat: string, username: string, type: string) => {
+  //나를 팔로우하는 사람
+  const followerList = useQuery({
+    queryKey: ['followerList'],
+    queryFn: async () => await getFollower(pat, username),
```

Comment:
함수에서 Promise를 바로 리턴하기 때문에 여기선 async await 를 써주지 않아도 돼요.


```suggestion
    queryKey: ['followerList'],
    queryFn: () => getFollower(pat, username),
```

Reply:
아하!! 그렇군요!! promise를 반환하면 async와 await가 필요없다는 것을 배워갑니다!!

Reply:
react query 같은 라이브러리를 공부할 때 어느정도 섬세하게 공부해야하는지 고민이었는데 학습하는 방법을 좀 더 확립할 수 있을 것 같습니다 감사합니다